wiki/People/The_Team/Beatmap_Nominators/Rules/id.md

```diff
@@ -0,0 +1,76 @@
+- **Pastikan tidak ada postingan mod dalam diskusi beatmap yang diabaikan sebelum menominasi sebuah beatmap.** Postingan tanpa balasan yang memadai dapat dianggap sebagai postingan yang diabaikan. Jika ragu, periksa kembali apakah Kode Etik Mapping dan Modding telah dilanggar di diskusi beatmap dan lakukanlah tindakan yang sesuai.
+
+### Kualifikasi
+
+- **Beatmap yang berbeda dari versi yang sama dari sebuah lagu tidak dapat dikualifikasikan di mode yang sama pada waktu yang bersamaan.** Versi yang sama dari sebuah lagu seperti potongan, perpanjangan, dan cover sederhana.
+- **Hanya 3 versi lagu yang berbeda yang dapat dikualifikasikan di mode yang sama pada waktu yang bersamaan.** Versi yang berbeda seperti remix, kompilasi, dan mashup.
+- **Jangan mengkualifikasi beatmap dari suatu mode dimana kamu bukan merupakan anggota penuh, kecuali jika sudah ada anggota penuh yang menominasinya.** Hal ini untuk memastikan bahwa seorang anggota nominator penuh selalu menjadi bagian dari proses untuk setiap mode.
+- **Jangan mengkualifikasi sebuah beatmap secara langsung tanpa komunikasi apapun dengan mapper.** Mapper tersebut mungkin telah melakukan perubahan sendiri setelah nominasi terakhir, jadi menominasi sebuah beatmapset tanpa berkomunikasi dengan mapper dapat menyebabkan diskualifikasi yang sia-sia.
+- **Setiap mode dalam sebuah beatmapset hybrid harus dinominasikan oleh dua nominator yang ahli sesuai dengan masing-masing mode.** Karena proses nominasi hybrid tidak didukung sepenuhnya, nominator yang ahli dalam beberapa mode tidak dapat menyetujui lebih dari satu mode.
+
+## Mereset nominasi
+
+- **Saat mereset nominasi dari sebuah beatmap, baris pertama dari postingan tersebut harus berisi alasan singkat dari reset tersebut.** Menautkan ke postingan saja tidak cukup, kamu harus mencoba meringkas postingan yang relevan. Dalam kasus diskualifikasi, hanya menulis bahwa itu diminta saja tidak cukup. Kamu harus menautkan semua masalah terkait yang diposting dalam diskusi beatmap kecuali jika terlalu banyak untuk diposting.
+- **Jika sebuah beatmap yang dinominasi mengandung masalah unrankable, nominasinya harus direset.** Hal ini juga berlaku jika kamu tidak berniat untuk menominasikan beatmapset.
+- **Pastikan Anda melihat versi terbaru dari sebuah beatmap sebelum mereset nominasi beatmap**.
+
+### Diskualifikasi
+
+- **Hanya anggota full yang dapat mendiskualifikasi beatmap, apapun alasannya.** Namun, anggota probationary masih dapat berpartisipasi dalam diskusi tersebut dan melaporkan beatmap seperti yang dilakukan orang lain.
+- **Diskualifikasi sebaiknya dilakukan untuk beatmap yang mempunyai masalah unrankable, atau jika pemilik beatmapset tersebut memintanya.**
+- **Permintaan diskualifikasi oleh pemilik beatmapset dapat segera diterapkan.** Ini termasuk diri kamu sendiri, misalnya kamu ingin melakukan dq sendiri.
```

Comment:
```suggestion
- **Permintaan diskualifikasi oleh pemilik beatmapset dapat segera diterapkan.** Ini termasuk diri kamu sendiri, misalnya kamu ingin melakukan diskualifikasi sendiri.
```
I am aware that EN ver uses `self-dq`, but uses `dq` here may cause confusion instead,

Reply:
ehh iya ya, awalnya sebenernya mau buat DQ huruf besar tapi jadinya huruf kecil entah mengapa. tapi bener juga, DQ kurang umum kalo buat orang awam.